### ISSUE TITLE
Questions about your paper 

### ISSUE THREAD

**Issue by Kieran31** [chenmingxiang110/AugNet#5]
Hello,

After reading your paper and trying your codes, I have several questions below.

1. On page 5 of your paper, you wrote 

> "In the above model, we use Resnet18 as the backbone and STL-10 unlabeled dataset (containing 100,000 images) as the training set."

But why your repo has the [res34_model_training_with_STL](https://github.com/chenmingxiang110/AugNet/blob/main/res34_model_training_with_STL.py)? Did a new experiment show that res34 outperformed res18? 

2. In table 6, you compared image retrieval quality among different image retrieval models and your model, did you also train all image retrieval models on Pokemon, Anime Face, and Human Sketch data? 

3. The model on ImageNet and the model on STL have two different output length (768 and 512), why did you specify different fully-connected layers on different dataset? Have you tried to make the feature vector smaller? 

4. why the loss is not compared each epoch? It looks like `n_pics * n_samples` is your "batch size". In each round, you picked `n_pics` original images and augmented `n_samples` times so your `Xs.shape == [n_pics * n_samples, 3, 224, 224]`. I see you compared the "cur loss" with best loss every five rounds, what's the purpose of this?

**Comment by chenmingxiang110** [comment]
Thank you for the questions.

1. For your first question, I apologize for the confusion. The files we provided are only examples of how we train models. As you can see from the script, we have tried our best to keep the code simple. If you are interested in reproducing the experiments shown in the paper, for example, the STL one, you can simply change the backbone to Net18 (we have just added it into the utils_torch.py file, please check out). The pokemon model and anime face model are trained in the same style. This avoids uploading duplicated models.

2. The models for Pokemon, Anime Face, and Human Sketch datasets are 3 separate models training on each dataset only.

3. Because ImageNet images have higher resolutions so we use a bigger model with a longer output length.

4. There is no particular reason for this. If it is on a large dataset, it might take days to train only one epoch. The model is saved every n rounds just in case something unfortunate happens.

**Comment by Kieran31** [comment]
Thanks @chenmingxiang110 for your explanation. It's very clear to me to understand your code and paper better.
